packages/paste-core/components/form/stories/index.stories.tsx

```diff
@@ -190,4 +190,33 @@ storiesOf('Components|Form', module)
         <FormHelperText>Info that helps a user with this field.</FormHelperText>
       </>
     );
+  })
+  .add('Textarea Options', () => {
```

Comment:
Can we get a story for every visual variant and state a textarea can appear in please? Just for VRT purposes.

Reply:
@SiTaggart added VRT stories.